src/components/SearchBar.vue

```diff
@@ -2,8 +2,8 @@
   <div class="search-bar-module__searchBar">
     <h3>Find your movie</h3>
     <div class="search-bar-module__searchInput">
-      <input type="text" v-model="searchQuery" @keyup.enter="updateFilteredMovies"/>
-      <button-component @click="updateFilteredMovies">Search</button-component>
+      <input type="text" v-model="searchQuery" @keyup.enter="submitSearch"/>
```

Comment:
There's a small UX problem with search functionality.
By pressing "search" it's expected to work depending on the "sort by" toggle - and it does work.
However, rearranging items after the search by toggling "sort by" should also be supported.